### ISSUE TITLE
Remove automatedTestEnabled flag from package.json

### ISSUE THREAD

**Issue by chrisklus** [phetsims/perennial#129]
@zepumph, @samreid, and I were investigating options for https://github.com/phetsims/phet-io/issues/1443#issuecomment-482761618, and noticed that the flag `automatedTestEnabled` might be redundant and should be removed. Couldn't we instead just add and remove repo names from active-runnables? There's only two usages of `automatedTestEnabled=false` (eliot and protein-synthesis).

**Comment by jonathanolson** [comment]
It seems like they should be things that are maintained, but not tested. Is that correct?

**Comment by samreid** [comment]
Decision from today's dev meeting.  We will remove eliot and protein-synthesis from active-runnables.

**Comment by samreid** [comment]
Removed in preceding commit.  I went to take the next step of eliminating the `automatedTestEnabled` flag from package.json, but it led me to `testable-phet-io` and `testable-runnables`, described in https://github.com/phetsims/perennial/blob/d9d0e182060c931c7b241e642ba9b02fa929ce18/data/README.md.  @zepumph do we still need these?  Or can they be eliminated for the same reason we want to eliminate `automatedTestEnabled`?

**Comment by zepumph** [comment]
I think the best way to proceed with this issue is to do 2 things:

1. Remove this flag
2. Think through the phet-io instrumentation life cycle, and create a way (if we think it necessary) to not ct test the phet-io features of a sim, but still be able to access the phet-io links from phetmarks. Right now they use two different lists, but they are differentiated based on `automatedTestEnabled`.
   * What if we made a flag like "phetioTestingOptOut" which you would have in addition to the phet-io supported brand until you were ready for that sim to be phet-io tested on CT.

**Comment by samreid** [comment]
This will take more time to work through, so I'm reducing priority relative to other high priority tasks.

**Comment by samreid** [comment]
I'm wondering how much of this was covered by https://github.com/phetsims/tandem/issues/191

**Comment by samreid** [comment]
Now that we have `phetioValidation` can we proceed to remove `automatedTestEnabled`?  @zepumph please confirm and reassign to me to proceed.

**Comment by zepumph** [comment]
Yes I think we are ready. We should remove `automatedTestEnabled` which then means that we remove support for and delete `testable-phet-io` and `testable-runnables`. Those usages in `listContinuousTests` can just use `phet-io` and `active-runnables`. I confirmed that the "testable-" data lists are exactly the same as their base list.

I confirmed that generate-data is up and running on bayes, so after altering `generateData.js`, that will need a restart:
* log onto phet-admin on bayes
* `pm2 restart `generate-data`

**Comment by samreid** [comment]
I ran the above steps, back to @zepumph for review.

**Comment by zepumph** [comment]
I got rid of the lists also. I think this is ready to close. Thanks @samreid!
